tpgtools/README.md

```diff
@@ -176,3 +176,33 @@ doc_hide:
 test_hide:
   - basic_workload.yaml
 ```
+
+## New Resource Guide (tpgtools only)
+
+This guide is written to document the process for adding a resource to the Google Terraform Provider (TPG) after it has been added to the [DCL](https://github.com/GoogleCloudPlatform/declarative-resource-client-library).
+
+### Adding Resource Overrides
+
+Every resource added via tpgtools needs an override file for every version it is available at. This file may be empty, but must exist. A resource available at GA (TPG) must also exist at beta (TPGB) and needs a corresponding override file at beta. These override files are often identical between versions. This file should exist at tpgtools/overrides/$PRODUCT_NAME/$VERSION/$RESOURCE.yaml. For example, [this override](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/tpgtools/overrides/assuredworkloads/beta/workload.yaml) exists for the product assuredworkloads resource workload at beta version.
+
+#### What should I add to the overrides file?
+
+For the first pass try nothing. If behavior is needed to change the resource's behavior in Terraform from the default DCL resource, then you can add these at a later date. Generally no overrides will be needed for a new resource, but they will be needed for resources that exist in mmv1 that are being converted to use tpgtools.
+
+### Adding Samples
+
+See [guide above](#samples) for adding samples.
+
+### Adding Additional Information to Handwritten/MMv1 Generated Files
```

Comment:
nit: "Additional Information..." feels optional. "Registering the Resource", maybe?